data/en/2020/sponsors.yml

```diff
@@ -8,3 +17,12 @@ items:
   - title: "Embecosm"
     image: "/images/sponsors/embecosm.png"
     url: "https://www.embecosm.com/"
+  - title: Global Foundries
+    image: /images/sponsors/global-foundries.png
+    url: http://www.globalfoundries.com/
+  - title: "Metrics"
+    image: "/images/sponsors/metrics.png"
```

Comment:
Was this image already there? You're not adding it in this PR.

Reply:
Yep! Was a sponsor in 2019, a return customer ;) https://github.com/openhwgroup/osdforum.org/blob/master/static/images/sponsors/metrics.png